components/Buttons/src/MDCButton.m

```diff
@@ -536,6 +536,10 @@ - (void)setBackgroundColor:(nullable UIColor *)backgroundColor {
   [self updateBackgroundColor];
 }
 
+- (UIColor *)backgroundColor {
```

Comment:
What's the difference between backgroundColorForState and backgroundColor in MDCButton?  In what cases is it preferred to call one over the other?

Reply:
In UIButton there are a whole bunch of readonly properties that do this, but they have a `current` prefix. I guess it's probably not worth a breaking change to keep that convention here, huh.

Reply:
Oh you're overriding UIView's backgroundColor huh